bin/configure_ccm_audit_chronicle.sh

```diff
@@ -69,6 +69,9 @@ logger_backend:
         roll_cycle: MINUTELY
         max_log_size: 1073741824 # 1GB
 
+whitelist_cache_validity_in_ms: 30000
```

Comment:
I propose to use the same values for our own whitelist cache as we're doing for the auth related caches.

And since we typically use these scripts for rather short test runs we should probably keep the values lower than you would in production, just to make the test runs more "interesting"

In other words, I propose to set these to 10000, 2000 (and 10000 or just leave it at default).

Reply:
ok